src/request.rs

```diff
@@ -220,6 +220,8 @@ pub enum Request {
         amount: Coins,
         /// The ID of the transaction.
         transaction_id: TransactionId,
+        /// If this transfer is a refund, this field contains the reason for it. Otherwise it is `None`.
```

Comment:
I would probably add just one bit saying something like "it should be always set to `None` for requests sent by clients"